modules/http/src/main/java/com/spotify/ffwd/http/HttpDecoder.java

```diff
@@ -87,4 +95,13 @@ private void postBatch(
             .writeAndFlush(new DefaultFullHttpResponse(HttpVersion.HTTP_1_1, HttpResponseStatus.OK))
             .addListener((ChannelFutureListener) future -> future.channel().close());
     }
+
+    private void getPing(
```

Comment:
params in and out are not used

Reply:
No, but it makes the methods handling requests consistent.